### PR TITLE
feat: Ze German Translation 🇩🇪

### PR DESCRIPTION
**Pull request by janl**

### CODE REVIEW

File: locales/de.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "Execute binaries from npm packages.\n%s": "Führt Programme aus, die von npm Paketen bereitgestellt werden.\n%s",
+  "Package to be installed.": "Das zu installierende Paket.",
+  "Location of the npm cache.": "Ort des npm Zwischenspeichers.",
+  "Skip installation if a package is missing.": "Überspringe die Installation falls ein Paket fehlt.",
+  "Path to user npmrc.": "Pfad zur Benuzter_innen npmrc.",
```
Comment (zkat) [inline locales/de.json:6]: Is this underscore a typo?

**Review by zkat — COMMENTED** [review]
Quick question otherwise lgtm

File: locales/de.json (hunk 1 above)
Comment (janl) [reply locales/de.json:6]: German is brutally gendered, so for more inclusive writing, we have to workaround with funky punctuation.

Here for the translation of `user` the underscore-version is a mix version of the male “Benutzer” + female “Benutzerinnen” + [Gender Gap](https://de.wikipedia.org/wiki/Gendergap_(Linguistik))(🇩🇪Wikipedia), where the underscore is the placeholder for all other genders.

An [alternative](https://de.wikipedia.org/wiki/Gendergap_(Linguistik)#Variationen) would be `Benutzer*innen`. The difference is mostly stylistic, although `_` is slightly more common as per my perception.

Also acceptable, although less inclusive (no space/star for non-binaries): `BenutzerInnen`.

**Comment by janl** [comment]
One other alternative would be to use the feminine form "Benutzerinnen" as a "generic feminine form", as you might imagine, it's usually done the other way around. This option is a bit more confrontational than the others, but totally valid too.

File: locales/de.json (hunk 1 above)
Comment (janl) [reply locales/de.json:6]: One other alternative would be to use the feminine form "Benutzerinnen" as a "generic feminine form", as you might imagine, it's usually done the other way around. This option is a bit more confrontational than the others, but totally valid too.

Comment (zkat) [reply locales/de.json:6]: Cool! Thanks for the explanation! I asked a couple of folks I was hanging out with and they weren't sure what it was about so I figured I'd ask.

TIL!

**Pull request merged**